apps/aecore/src/aec_block_key_candidate.erl

```diff
@@ -34,13 +34,13 @@ create(Block, Beneficiary) ->
                            {ok, aec_blocks:block()} | {error, term()}.
 adjust_target(Block, AdjHeaders) ->
     Header = aec_blocks:to_header(Block),
-    DeltaHeight = aec_governance:key_blocks_to_check_difficulty_count(),
+    DeltaHeight = aec_governance:key_blocks_to_check_difficulty_count() + 1,
```

Comment:
we have mysterious +1 (or -1) all over the place. Why not to bump it in governance?

Reply:
I believe this is because of the difference between validating the target and generating next target, so it probably cannot be fixed in one place.

Reply:
Yes, exactly, we are using the targets of the last 17 blocks, but also the solvetimes of the last 17 blocks. However, the latter require 18 blocks to compute... I prefer the constant to be 17 then since that is the `N` in the algorithm.